virtualization/windowscontainers/deployment/docker_windows.md

```diff
@@ -169,7 +169,7 @@ Now, on your machine, open a PowerShell or CMD session, and run the Docker comma
 .\docker.exe -H tcp://<ip address of Nano Server>:2375
 ```
 
-For example, if you would like to see the available images: 
+For example, if you would like to see the available images:
 
 ```powershell
 .\docker.exe -H tcp://<ip address of Nano Server>:2375 images
```

Comment:
That should still be fine - use npipe:////servername/pipe/docker_engine. There should not be any explicit 2375 listening on TCP regardless of nano/servercore, local or remote.